config/initializers/pg_search.rb

```diff
@@ -14,7 +14,9 @@
         MinWords: 5,
         MaxWords: 10
       }
-    }
+    },
   },
+  # https://github.com/Casecommons/pg_search#ranked_by-choosing-a-ranking-algorithm
+  ranked_by: "LENGTH(content)",
```

Comment:
From what I understand, this is not taking the search scores into account but purely works on the document length. I'd have expected something like 

```ruby
:tsearch * length(content)
```

or similar.

Reply:
yeah, that was just a quick experiment. I also tried using `:tsearch` and the other algorithms. The point is you can't use `pg_search` to rank based on patterns on multiple columns which is what the core of the ES functionality is.